docs/MO_DG/IR_and_opsets.md

```diff
@@ -14,33 +12,33 @@ Here is an illustration of a small graph representing a model that consists of a
 ![](img/small_IR_graph_demonstration.png)
 
 Vertices in the graph represent layers or operation instances, like convolution, pooling or element-wise operations with tensors.
-Layer and operation terms are used interchangeably along the OpenVINO™ documentation and define how input data is processed to produce output data for a node in a graph.
+Layer and operation terms are used interchangeably along the OpenVINO documentation and define how input data is processed to produce output data for a node in a graph.
 An operation node in a graph may consume data at one or multiple input ports.
-For example, element-wise addition operation has two input ports which accepts tensors that are added together.
-Some operations don't have any input ports, for example Const operation which knowns the data to be produced without any input.
+For example, element-wise addition operation has two input ports which accept tensors that are added together.
+Some operations do not have any input ports, for example *`Const`* operation, which knows the data to be produced without any input.
 An edge between operations represent data flow or data dependency implied from one operation node to another operation node.
 
-Each operation produces data on one or multiple output ports. For example, convolution produces output tensor with activations at a single output port. Split operation usually has multiple output ports each producing part of an input tensor.
+Each operation produces data on one or multiple output ports. For example, convolution produces output tensor with activations at a single output port. Split operation usually has multiple output ports, each producing part of an input tensor.
 
 Depending on a deep learning framework, the graph can also contain extra nodes that explicitly represent tensors between operations.
 In such representations, operation nodes are not connected directly to each other, rather using data nodes as intermediate stops for data flow.
 If data nodes are not used, the produced data is associated with an output port of a corresponding operation node that produces the data.
 
 A set of various operations used in a network is usually fixed for each deep learning framework.
 It determines expressiveness and level of representation available in that framework.
-It may happen that a network that can be represented in one framework is hard or impossible to be represented in another one or should use significantly different graph because operation sets used in those two frameworks do not match.
+A network that can be represented in one framework is sometimes hard or impossible to be represented in another one or should use significantly different graph because operation sets used in those two frameworks do not match.
```

Comment:
```suggestion
Sometimes, a network that can be represented in one framework is hard or impossible to be represented in another one or should use significantly different graph, because operation sets used in those two frameworks do not match.
```